presidio_evaluator/models/text_analytics_wrapper.py

```diff
@@ -23,7 +23,8 @@ def __init__(
         :param ta_client: object of type TextAnalyticsClient
```

Comment:
changes here:
- formatting
- adding `batch_predict` to conform with base class